src/library/Cardano/CoinSelection/Fee.hs

```diff
@@ -305,13 +300,14 @@ reduceChangeOutputs threshold (Fee totalFee) changeOutputs
             [] -> []
   where
     payFee :: (Fee, Coin) -> Coin
-    payFee (Fee f, Coin c) = Coin (c - f)
+    payFee (Fee f, Coin c) = Coin $
+        fromMaybe SN.zero ((c `SN.sub`) =<< SN.fromIntegral f)
 
     positiveChangeOutputs :: [Coin]
-    positiveChangeOutputs = filter (> Coin 0) changeOutputs
+    positiveChangeOutputs =
+        Coin <$> filter SN.isPositive (unCoin <$> changeOutputs)
```

Comment:
I find the term `Positive` actually a lot less clearer than a mathematical inequality `> 0`. The reason for this is that maths are unambiguous (in this context at least) whereas 'Positive' requires an careful understanding of the English language. why I am pointing that out here is because, while `Positive` in English means `> 0`, `Positive` in French means `>= 0` and it's probably the case in other languages too. 

When possible, sticking to mathematical notation may be better.

Reply:
> I find the term `Positive` actually a lot less clearer than a mathematical inequality `> 0`. The reason for this is that maths are unambiguous (in this context at least) whereas 'Positive' requires an careful understanding of the English language.

Thanks. I wasn't aware that this distinction existed in French. After researching this, I discovered the term "strictement positif", which seems similar to the distinction between a subset and a "strict" subset.

For the avoidance of any doubt, I've replaced these with `> SN.zero`.

Fixed in e98fd62a95b743dc4b821c2be581861ee50d0978.